engine/admin/live-restore.md

```diff
@@ -71,4 +71,7 @@ You can modify the kernel's buffer size by changing `/proc/sys/fs/pipe-max-size`
 
 The live restore option is not compatible with Docker Engine swarm mode. When
 the Docker Engine runs in swarm mode, the orchestration feature manages tasks
-and keeps containers running according to a service specification.
\ No newline at end of file
+and keeps containers running according to a service specification.
+
+ **Note**: The live restore option is only supported on Linux.
```

Comment:
I did the change, but i did not see any difference while previewing the .md file in github. Can you check if its fine?